technical/consensus/consensus.py

```diff
@@ -536,6 +571,86 @@ def evaluate_adx(self, period=14, prefix="momentum", impact_buy=1, impact_sell=1
         name = f"{prefix}_{period}"
         dataframe[name] = ta.ADX(dataframe, timeperiod=period)
 
-        dataframe.loc[((dataframe[name] > 25)), f"buy_{name}"] = 1 * impact_buy
+        # We also need to use PLUS_DI and MINUS_DI to be able to detect if we should buy or sell
+        dataframe[f"{name}_plus_di"] = ta.PLUS_DI(dataframe, timeperiod=period)
+        dataframe[f"{name}_minus_di"] = ta.MINUS_DI(dataframe, timeperiod=period)
```

Comment:
sorry for the patchwork Review - i only noticed this now (Right before merging)

This section is not mentioned in the docstring ... which only says "evaluates the adx" ... but it should at least mention that it'll also look at plus/minusDI to detect trend direction.